src/contracts/core/RewardsCoordinator.sol

```diff
@@ -509,8 +554,22 @@ contract RewardsCoordinator is
 
     /// @notice the commission for a specific operator for a specific avs
     /// NOTE: Currently unused and simply returns the globalOperatorCommissionBips value but will be used in future release
-    function operatorCommissionBips(address operator, address avs) external view returns (uint16) {
-        return globalOperatorCommissionBips;
+    function operatorCommissionBips(
```

Comment:
Nit: `getCurrentOperatorCommissionBips`?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/627/commits/68478ee59925e8e2cbb5fb1bfc348079fe6cb614
updated to `getOperatorCommissionBips` and `setOperatorCommissionBips`